### ISSUE TITLE
Support v3.0.0 of Prettier

### ISSUE THREAD

**Issue by softwarespot** [npetruzzelli/prettier-config-standard#8]
This is the following output when using the latest version of Prettier. Is it possible to update the peer dependency to support the latest version? Happy to provide more information. Thanks
```
npm ERR! Could not resolve dependency:
npm ERR! peer prettier@"^2.4.0" from prettier-config-standard@5.0.0
npm ERR! node_modules/prettier-config-standard
npm ERR!   dev prettier-config-standard@"^5.0.0" from the root project
npm ERR!   peer prettier-config-standard@">=1.0.0" from eslint-config-prettier-standard@4.0.1
npm ERR!   node_modules/eslint-config-prettier-standard
npm ERR!     dev eslint-config-prettier-standard@"^4.0.1" from the root project
npm ERR! 
npm ERR! Conflicting peer dependency: prettier@2.8.8
npm ERR! node_modules/prettier
npm ERR!   peer prettier@"^2.4.0" from prettier-config-standard@5.0.0
npm ERR!   node_modules/prettier-config-standard
npm ERR!     dev prettier-config-standard@"^5.0.0" from the root project
npm ERR!     peer prettier-config-standard@">=1.0.0" from eslint-config-prettier-standard@4.0.1
npm ERR!     node_modules/eslint-config-prettier-standard
npm ERR!       dev eslint-config-prettier-standard@"^4.0.1" from the root project
```

**Comment by jonathanlinat** [comment]
Here, the same output from `pnpm`:

![image](https://github.com/npetruzzelli/prettier-config-standard/assets/14064112/f67fd279-6578-4465-95eb-ba34003cd485)

**Comment by neftaly** [comment]
yarn: `... provides prettier (p77055) with version 3.0.0, which doesn't satisfy what prettier-config-standard requests`

Looks like breaking changes are: 
* ecma modules (not relevant)
* plugin api (not used?)
* trailingComma defaults to 'all' (won't affect this repo).

<https://prettier.io/blog/2023/07/05/3.0.0.html>

**Comment by npetruzzelli** [comment]
Thank you for opening this issue here and the related one in [`eslint-config-prettier-standard`](https://github.com/npetruzzelli/eslint-config-prettier-standard/issues/17)

I will look into it as soon as possible. 

<details>

I don't expect it to be difficult, I just need the time to do it, which is in short supply lately.

If you don't see an update by August 16th, it is possible I did that human thing (forgetting about it) and I wouldn't begrudge anyone asking "is there an update on this" as a reminder, but I'll try to make sure it doesn't come to that.

</details>

**Comment by npetruzzelli** [comment]
Closed by #9

Please leave additional comments if you feel that I missed something!
